src/pages/ColoredManga/main.ts

```diff
@@ -0,0 +1,65 @@
+import { pageInterface } from '../pageInterface';
```

Comment:
Not all Chapters are recognized

https://coloredmanga.com/mangas/medaka-box/volume-12/box-106/
https://coloredmanga.com/mangas/spy-x-family-colour/volume-10-bw/mission-67-2-bw/
https://coloredmanga.com/mangas/eng-rurouni-kenshin/volume-28/act-255-finale/